tests/test_helpers.py

```diff
@@ -348,4 +348,10 @@ def test_vtk_points_deep_shallow():
     assert not np.array_equal(vtk_points.GetPoint(0), points[0])
     assert np.array_equal(points[0], [0.0, 0.0, 0.0])
 
-    # TODO: test force_float
+
+@pytest.mark.parametrize("force_float,expected_data_type", [(False, np.int64), (True, np.float32)])
+def test_vtk_points_force_float(force_float, expected_data_type):
+    np_points = np.array([[1, 2, 3]], dtype=np.int64)
+    pset = pyvista.PointSet(np_points, force_float=force_float)
```

Comment:
This test is fine after the intricacies are worked out.  My comment from the previous PR was that we should directly test `vtk_points` to fully close #3333.  For example this line would be `vtk_points = pyvista.vtk_points(np_points, force_float=force_float)` and test that this object has the correct type.

Reply:
By bad, I moved the code and somehow mixed the content of the two files.

It's now fixed.